samples/apps/copilot-chat-app/webapi/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Wouldn't this file be more at home in the config subfolder?

Reply:
Loading configurations is usually the domain of the service framework. ASP.NET has default logic for loading appsettings that we are overriding here, so I thought it would be best kept near where we do other things with IServiceProvider. That and /config folder seems to be meant to have our configuration models and this code doesn't use any of the models.